sunpy/time/tests/test_time.py

```diff
@@ -12,119 +13,133 @@
 
-    assert isinstance(dt, datetime)
+    assert isinstance(dt, astropy.time.Time)
     assert dt == LANDING
 
 
 def test_parse_time_pandas_series():
     inputs = [datetime(2012, 1, i) for i in range(1, 13)]
     ind = pandas.Series(inputs)
+    as_inps = ap.Time(inputs)
 
     dts = parse_time(ind)
 
-    assert isinstance(dts, np.ndarray)
-    assert all([isinstance(dt, datetime) for dt in dts])
-    assert list(dts) == inputs
+    assert isinstance(dts, astropy.time.Time)
+    assert np.all(dts == as_inps)
+
+
+def test_parse_time_pandas_series_2():
+    inputs = [[datetime(2012, 1, 1, 0, 0), datetime(2012, 1, 2, 0, 0)],
+              [datetime(2012, 1, 3, 0, 0), datetime(2012, 1, 4, 0, 0)]]
+    ind = pandas.Series(inputs)
+    as_inps = ap.Time(inputs)
+
+    apts = parse_time(ind)
+
+    assert isinstance(apts, astropy.time.Time)
+    assert np.all(apts == as_inps)
+    assert apts.shape == as_inps.shape
 
 
 def test_parse_time_pandas_index():
     inputs = [datetime(2012, 1, i) for i in range(1, 13)]
     ind = pandas.DatetimeIndex(inputs)
+    as_inps = ap.Time(inputs)
 
     dts = parse_time(ind)
 
-    assert isinstance(dts, np.ndarray)
-    assert all([isinstance(dt, datetime) for dt in dts])
-    assert list(dts) == inputs
+    assert isinstance(dts, astropy.time.Time)
+    assert np.all(dts == as_inps)
 
 
 def test_parse_time_numpy_date():
     inputs = np.arange('2005-02', '2005-03', dtype='datetime64[D]')
 
     dts = parse_time(inputs)
 
-    assert isinstance(dts, np.ndarray)
-    assert all([isinstance(dt, datetime) for dt in dts])
+    assert isinstance(dts, astropy.time.Time)
 
 
 def test_parse_time_numpy_datetime():
     inputs = np.arange('2005-02-01T00', '2005-02-01T10', dtype='datetime64')
 
     dts = parse_time(inputs)
 
-    assert isinstance(dts, np.ndarray)
-    assert all([isinstance(dt, datetime) for dt in dts])
+    assert isinstance(dts, astropy.time.Time)
 
 
 def test_parse_time_individual_numpy_datetime():
     dt64 = np.datetime64('2005-02-01T00')
     dt = parse_time(dt64)
 
-    assert isinstance(dt, datetime)
+    assert isinstance(dt, astropy.time.Time)
+    assert dt == ap.Time('2005-02-01', format='isot')
 
 
 def test_parse_time_numpy_datetime_timezone():
     dt64 = np.datetime64('2014-02-07T16:47:51-0500')
     dt = parse_time(dt64)
 
-    assert dt == datetime(2014, 2, 7, 21, 47, 51)
+    assert dt == ap.Time('2014-02-07T21:47:51', format='isot')
 
 
 def test_parse_time_numpy_datetime_ns():
     dt64 = np.datetime64('2014-02-07T16:47:51.008288000')
     dt = parse_time(dt64)
 
-    assert dt == datetime(2014, 2, 7, 16, 47, 51, 8288)
+    assert dt == ap.Time('2014-02-07T16:47:51.008288000', format='isot')
 
     dt64 = np.datetime64('2014-02-07T16:47:51.008288123')
     dt = parse_time(dt64)
 
-    assert dt == datetime(2014, 2, 7, 16, 47, 51, 8288)
+    assert dt == ap.Time('2014-02-07T16:47:51.008288123', format='isot')
 
 
 def test_parse_time_numpy_datetime_round():
```

Comment:
Should the name be changed since it's not rounding anymore?

Reply:
yeah I think so